man/git-count.md

```diff
@@ -23,6 +24,12 @@ git-count(1) -- Show commit count
     $ git count
 
     total 1844
+ 
+ Output full commits total:
```

Comment:
Let's add more words to explain the difference between the regular total and the full total.

Reply:
The regular total count from current commit to the first one and the full total count all commits available at repository.